src/styles/calendar.scss

```
@@ -0,0 +1,51 @@
+.calendar {
+  margin: auto;
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  gap: $gap-size;
+  padding: 20px;
+
+  &__day {
+    width: $cell-size;
+    height: $cell-size;
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    border: 1px solid #000;
+    border-radius: 10%;
+    transition: (0.5s);
```

```suggestion
    transition: color 0.5s, background-color 0.5s, transform: 0.5s;
```
Specify props and move 0.5s to the variable